js/components/DeploymentComponent.jsx

```diff
@@ -81,27 +99,27 @@ var DeploymentComponent = React.createClass({
       // Set `title` on cells that potentially overflow so hovering on the
       // cells will reveal their full contents.
       <tr>
-        <td className="overflow-ellipsis" title={model.get("id")}>
-          {model.get("id")}
+        <td className="overflow-ellipsis" title={model.id}>
+          {model.id}
         </td>
         <td>
           <ul className="list-unstyled">
-            {model.get("currentActions").map(function (action) {
+            {lazy(model.currentActions).map(function (action) {
               return <li key={action.app}>{action.app}</li>;
-            })}
+            }).value()}
           </ul>
         </td>
         <td>
           <ul className="list-unstyled">
-            {model.get("currentActions").map(function (action) {
+            {lazy(model.currentActions).map(function (action) {
               return <li key={action.app}>{action.action}</li>;
-            })}
+            }).value()}
```

Comment:
Because we are using Lazy.js, the ```.value()``` triggers the evaluation of the so-called 'sequence'.
For better understanding in this case, instead of a .value(), we could also use a ```.toArray()``` to lazy evaluate the sequence. ```.value()``` is 'sugar'.

Look at ```That's where Lazy.js comes in! ```:
http://danieltao.com/lazy.js/